### DISCUSSION TITLE
What system does this site use?

### DISCUSSION THREAD

**Discussion by bedipavel** [ophian/styx#50] [General]
Don't know what system this website uses? https://www.eida.cz/
In the footer it says GC-StyX 1.3.42.0. Is this your system or another? Don't know when to get this system?
Thank you.

**Comment by ophian** [comment]
I don't know too and have nothing to do with it. I think it is something privatly homebrew which has some generic matches with (Serendipity Styx) blog(s), but in a more common mean. I assume you could build that with Serendipity Styx too. 
The Eida is quite funny, at least in the translation.
Converted to discussion.
